src/popup/components/InviteItem.vue

```diff
@@ -38,7 +38,7 @@
     </div>
     <template v-else>
       <InputAmount
-        v-model="topUpAmount"
+        v-model.number="topUpAmount"
```

Comment:
The number approach will break the long numbers logic we have, and we will lose precision.